core/kernel/initcall.c

```diff
@@ -7,6 +7,9 @@
 #include <initcall.h>
 #include <trace.h>
 #include <kernel/linker.h>
+#ifdef ARM64
+#include <arm64.h>
```

Comment:
```patch
+#include <arm.h>
 #include <initcall.h>
 #include <trace.h>
 #include <kernel/linker.h>
-#ifdef ARM64
-#include <arm64.h>
-#endif
```

By the way,  _kernel/linker.h_ should be reordered but that out of the scope of this P-R:
```c
#include <arm.h>
#include <initcall.h>
#include <kernel/linker.h>
#include <trace.h>
```

Reply:
Fixed